MiquelRaga_Setup.yaml

```
@@ -0,0 +1,167 @@
+version: 1.3.2
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)